tests/components/bond/test_cover.py

```diff
@@ -33,14 +38,14 @@ async def test_entity_registry(hass: core.HomeAssistant):
     await setup_platform(hass, COVER_DOMAIN, shades("name-1"))
 
     registry: EntityRegistry = await hass.helpers.entity_registry.async_get_registry()
-    assert [key for key in registry.entities.keys()] == ["cover.name_1"]
+    assert [key for key in registry.entities] == ["cover.name_1"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)